### PR TITLE
improved README

### PR DESCRIPTION
**Pull request by erezmus**
- Renamed the "Secrets Management" to Admin
- Added section about AWS credentials
- Setup section that is developer specific
- Added section to fetch current terraform state

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +20,34 @@ They run inside the default VPC for `us-east-1` and are based on a minimal Ubunt
 
 DNS entries for `libero.pub` and similar domains are managed under this AWS account, hence here.
 
+## Setup
+
+This repository is public, hence secrets are encrypted by default. `git-crypt` and optionally `gpg` are used to unlock the contents to be able to use the repository contents.
+
+`git-crypt` encrypts data with a symmetric key, and places encrypted versions of that key in `.git-crypt`; one value for each public key of the users that should be able to access that data. It is easy to add new users, more complex to remove them.
+
+Once your public key has been added you can unlock the repository (see Admin section below for how), you can decrypt the repository:
```
Comment (giorgiosironi) [inline README.md]: make `Admin` a link?

File: README.md [hunk 2]
```diff
@@ -20,8 +20,34 @@ They run inside the default VPC for `us-east-1` and are based on a minimal Ubunt
...
+git-crypt unlock
+```
+
 ## Terraform usage
 
+You will need to set up AWS credentials in your ~/.aws/credentials file:
+
+```
+[libero]
+aws_acess_key=<access key id>
```
Comment (giorgiosironi) [inline README.md]: `aws_access_key_id=...`

File: README.md [hunk 3]
```diff
@@ -20,8 +20,34 @@ They run inside the default VPC for `us-east-1` and are based on a minimal Ubunt
...
+aws_secret_access_key=<secret access key>
+```
+
+Then set your current AWS profile:
+```bash
+$ export $AWS_PROFILE=libero
```
Comment (giorgiosironi) [inline README.md]: just `export AWS_PROFILE=libero`

File: README.md [hunk 4]
```diff
@@ -33,60 +59,68 @@ terraform plan -var env=$ENVIRONMENT_NAME -out=my.plan
 terraform apply my.plan
 ```
 
-Update the environment:
+### Fetch current environment state
+To get the state of an existing environment (here named `unstable`):
```
Comment (giorgiosironi) [inline README.md]: badly needed  as a section :+1:

File: README.md [hunk 5]
```diff
@@ -33,60 +59,68 @@ terraform plan -var env=$ENVIRONMENT_NAME -out=my.plan
...
 
+```bash
+ENVIRONMENT_NAME=unstable
+cd tf/
```
Comment (giorgiosironi) [inline README.md]: might want to `rm -rf .terraform` after this just in case there is something else set up

Comment (giorgiosironi) [reply README.md]: (at least that's what the build does)

File: README.md [hunk 6]
```diff
@@ -33,60 +59,68 @@ terraform plan -var env=$ENVIRONMENT_NAME -out=my.plan
...
+terraform init --backend-config="key=$ENVIRONMENT_NAME/terraform.tfstate"
 ```
+
+### Update an environment
+
+Update the environment:
```
Comment (giorgiosironi) [inline README.md]: add a comment on how normally `apply` should not be executed, as it's done via CI on merge

**Comment by diversemix** [comment]
@GiancarloFusiello ... I suggest we go through this as part of the hand over tomorrow morning.

/cc @giorgiosironi @erezmus @Cooryd @will-byrne @nlisgo @thewilkybarkid

**Pull request merged**